src/profile-logic/marker-data.js

```diff
@@ -138,7 +138,7 @@ export function getSearchFilteredMarkerIndexes(
       if (data.type === 'FileIO') {
         const { filename, operation, source, threadId } = data;
         if (
-          searchRegExp.test(filename) ||
+          searchRegExp.test(filename || '') ||
```

Comment:
Flow wants this.
I haven't checked because I didn't have a profile with that, but I believe that without this change, if we search for "undef" this would find the FileIO markers without filenames 😂.

Reply:
@fqueze if you can double check my assumption here, that would be nice :-)

Reply:
Your guess is correct: https://share.firefox.dev/37EkIPN